SignalServiceKit/src/Messages/Interactions/TSQuotedMessage.h

```diff
@@ -2,11 +2,47 @@
 //  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
 //
 
+#import <Mantle/MTLModel.h>
 #import <SignalServiceKit/TSYapDatabaseObject.h>
 
 NS_ASSUME_NONNULL_BEGIN
 
-@interface TSQuotedMessage : TSYapDatabaseObject
+@class OWSSignalServiceProtosDataMessage;
+@class OWSSignalServiceProtosEnvelope;
+@class TSAttachment;
+@class TSAttachmentStream;
+@class TSQuotedMessage;
+@class TSThread;
+@class YapDatabaseReadWriteTransaction;
+
+@interface OWSAttachmentInfo: MTLModel
```

Comment:
Rather than having an array of attachmentId's and a dictionary of `(attachmentId) -> sourceFileName` I made a small serializable model class to wrap that information.